### DISCUSSION TITLE
Deployment fails for Chirpy-Starter template due to missing assets

### DISCUSSION THREAD

**Discussion by openwiretech** [cotes2020/jekyll-theme-chirpy#1659] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Just chatting

### Description

Hi all,

I used the Chirpy-Start template and created a basic site to be used on git hub pages. When I push to my repo the deployment fails on "Test Site" because its complaining about missing favicons. I checked in "_sites" and the assets are there....not sure what the linking errors are

example:

```
For the Links > Internal check, the following failures were found:
internally linking to /assets/img/favicons/apple-touch-icon.png, which does not exist
```

**Comment by minhtuannn** [comment]
have you found the solution?

**Reply by shu-keep-moving** [reply]
What about you? I've encountered the same problem too.

**Comment by ys010** [comment]
got the same issue

**Comment by aaronzhengfc** [comment]
got the same issue

**Reply by SoumyaK4** [reply]
@Aaron-zheng share your repo link
